dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/MyRequestRetryHandler.java

```diff
@@ -0,0 +1,45 @@
+package com.dotcms.content.elasticsearch.util;
+
+import org.apache.http.HttpEntityEnclosingRequest;
+import org.apache.http.HttpRequest;
+import org.apache.http.client.HttpRequestRetryHandler;
+import org.apache.http.client.protocol.HttpClientContext;
+import org.apache.http.conn.ConnectTimeoutException;
+import org.apache.http.protocol.HttpContext;
+
+import javax.net.ssl.SSLException;
+import java.io.IOException;
+import java.io.InterruptedIOException;
+import java.net.UnknownHostException;
+
+public class MyRequestRetryHandler implements HttpRequestRetryHandler {
+    @Override
+    public boolean retryRequest(
+            IOException exception,
+            int executionCount,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'executionCount' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5121287)